### ISSUE TITLE
Interpretation of FlexRay Signals form a mf4 measurement file

### ISSUE THREAD

**Issue by Matts711** [danielhrisca/asammdf#496]
Hello together,

in some issue forums this topic has already discussed, it is about reading FlexRay Signals from a mf4 file. But the answers was not really helpful. 

I have a:
- mf4 measurement file with only flexray channels like: 'FLX_Frame.ID', 'FLX_Frame.Cycle', 'FLX_Frame.FlxChannel', ... etc.
- arxml file 

If I convert the mf4 file in Vector CANoe with the arxml to mdf file i can read in python the FlexRay signals "VehSpeed" or "YawAngel" ... etc.

**My question:** 
Is it today possible with the libray "asammdf" to include the arxml and get the FlexRay signals direct from mf4? 
Maybe convert in python mf4 to mdf with include arxml?

If it possible, how i can code it? 

Thank you very much in advance.

Best regards,
Matts

**Comment by danielhrisca** [comment]
Hello @Matts711 see #273 Flexray decoding is not possible now because there is no Fibex parser available

**Comment by Matts711** [comment]
Hey @danielhrisca thank you for your answer. 

I read all your comments on other issue forums here. 

I've tried it the last few days and only got the determine uncoded sample values ​​of the individual channels and the measurement time. **I saw that it is currently not possible to read FlexRay.**

Now i will decode the mf4 file with Vector CANoe and import the decode mf4 file in python.  After CANoe all signals are readable :)
